contrib/util/installScripts/cvsDemoInstall

```diff
@@ -98,7 +98,6 @@ chmod 666 $OPENEMR/sites/default/sqlconf.php
 chown -R apache:apache $OPENEMR/sites/default/documents
```

Comment:
This entire file and the entire contrib/util/installScripts/cvsDemoMisc directory can be removed.